agent/acs/handler/acs_handler.go

```diff
@@ -425,6 +416,22 @@ func anyMessageHandler(timer ttime.Timer) func(interface{}) {
 	}
 }
 
+func shouldReconnectWithoutBackoff(acsError error) bool {
+	if acsError == nil || acsError == io.EOF {
+		return true
+	}
+
+	return false
+}
+
+func isInactiveInstanceError(acsError error) bool {
+	if acsError != nil && strings.HasPrefix(acsError.Error(), "InactiveInstanceException:") {
```

Comment:
You can simplify this to:

```go
return acsError != nil && strings.HasPrefix(acsError.Error(), "InactiveInstanceException:")
```

Also, can you move "InactiveInstanceException" to a constant?